### ISSUE TITLE
Markdown 2 format bad for copying code blocks

### ISSUE THREAD

**Issue by zach-is-my-name** [Nemoden/gogpt#8]
Markdown2 draws a verticle bar in the markdown output.  Getting around this now by pasting to vim and visual block selecting and deleting the bar.  
    
 My prior issue #7 asks about toggling the output method to experiement with other output formats.  
    
    ┃ mkdir transformed_files && \
    ┃ for file in *; do \
    ┃     if [[ "$file" == *\ * ]]; then \
    ┃         new_file=$(echo "$file" | sed 's/ /_/g'); \
    ┃         cp "$file" "transformed_files/$new_file"; \
    ┃     else \
    ┃         cp "$file" transformed_files/; \
    ┃     fi; \
    ┃ done

**Comment by Nemoden** [comment]
@zach-is-my-name yep, unfortunately, the underlying markdown renderer doesn't allow styling options.

The best way is honestly to not use markdown formatters for now as they are still experimental, and even if they were not, there is not much I can do about the vertical line anyway which is not a hack. To properly solve this I'd need to implement formatting options into the underlying library by either forking it or doing a PR request and merging it into the lib.

I definitely will be looking to improve markdown renderers, but I'd need to properly assess possible solutions and pick what makes most sense, and even then the fix isn't the most straightforward thing, so it is not happening this month (April) at least 😬

So for now it's out of scope, unfortunately. 

What I can undertake right now is contacting authors of the markdown rendering lib and ask them if they are looking to support different styles / coloschemes etc, and I'll post here as soon as I hear from them.

**Comment by zach-is-my-name** [comment]
Thanks for the reply.  I understand.  For the future perhaps there's
something that can be done with the Charm suite
https://github.com/charmbracelet.  Until then gogpt is still my favorite
gpt cli tool.
